erpnext/setup/doctype/party_import/party_importer.py

```
@@ -0,0 +1,1140 @@
+
+		party_docname, party_created = self._process_party(group)
+		frappe.db.commit()
+
+		errors = []
+
+		address_created = False
+		for addr_row in group.address_rows:
+			try:
+				self._process_address(addr_row, party_docname, party_name)
+				address_created = True
+			except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)